superset/examples/helpers.py

```diff
@@ -41,7 +41,7 @@
 misc_dash_slices: Set[str] = set()  # slices assembled in a 'Misc Chart' dashboard
 
 
-def update_slice_ids(layout_dict, slices):
+def update_slice_ids(layout_dict: Dict[Any, Any], slices: List[Slice]) -> None:
```

Comment:
Not necessary a change that needs to be done here, but with proper typing, we don't necessarily need to include types in names as much in the future. Here `layout_dict` -> `layout`.

Reply:
@villebro I agree. I've never been a fan of variables the `_list`, `_dict`, etc. suffixes.